AzureCalling/Controllers/BottomDrawerViewController.swift

```diff
@@ -5,20 +5,23 @@
 
 import UIKit
 
-class BottomDrawerViewController: UIViewController {
+class BottomDrawerViewController: UIViewController, UIGestureRecognizerDelegate {
 
     // MARK: Properties
 
     private var tableView: UITableView!
     private var bottomDrawerDataSource: BottomDrawerDataSource?
+    private var allowRowSelection: Bool = false
 
     required init?(coder aDecoder: NSCoder) {
         super.init(coder: aDecoder)
     }
 
-    init(dataSource: BottomDrawerDataSource) {
+    init(dataSource: BottomDrawerDataSource, allowsSelection: Bool) {
```

Comment:
Can you make the `allowsSelection` variable to have default value `allowsSelection: Bool = false`. So only function calls that needs it enabled need to specify the parameter? (Or vice versa make the default behaviour `allowsSelection: Bool = true`)

Reply:
For sure, default value `false` makes sense since we want the base functionality of a bottom drawer to be just a list